src/data/injections.js

```diff
@@ -540,19 +508,19 @@ const AVAILABLE_INJECTIONS = [
         "*://*.avanabayview.com/*", // #118617
         "*://*.breakpointeandcoronado.com/*", // #117735
         "*://*.courtsatspringmill.com/*", // #128404
+        "*://*.fieldstoneamherst.com/*", // #131703
```

Comment:
I think this is supposed to be 132974, yes?

Reply:
good catch, thanks!